src/select/typeahead/TypeaheadSelect.tsx

```diff
@@ -187,6 +189,19 @@ function TypeaheadSelect({
     }
   }
 
+  function handleKeyDown(event: React.KeyboardEvent<HTMLInputElement>) {
+    const {key} = event;
+
+    if (
+      key === KEYBOARD_EVENT_KEY.ESCAPE &&
```

Comment:
```suggestion
      key === KEYBOARD_EVENT_KEY.BACKSPACE &&
```
I think there is a mistake